### DISCUSSION TITLE
2019.7-2020.7编程年记 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#183] [General]
# 2019.7-2020.7编程年记 - 愧怍的小站

前言

https://kuizuo.cn/2019.7-2020.7-log#comments

**Comment by fjqisba** [comment]
捕获一名易语言大手子

**Reply by kuizuo** [reply]
距离我卸载易语言已经过去快1年了，易语言几乎已经退坑了
